e2e/support/commands/visibility/findByTextEnsureVisible.ts

```diff
@@ -0,0 +1,15 @@
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      findByTextEnsureVisible(
+        text: string,
+      ): Cypress.Chainable<JQuery<HTMLElement>>;
+    }
+  }
+}
+
+Cypress.Commands.add("findByTextEnsureVisible", text => {
+  cy.findByText(text).should("be.visible");
```

Comment:
TBH, I never knew (or already forgot) this util exists. I'd usually find the element and call `.should("be.visible")` manually. That way you could possibly use any of the methods to find your elements and don't have to be limited to just `findByText`.

I'm not saying we should remove this util since it's being used in many places. I just want to share my opinion.